explorer/app/components/Section/components/SectionContent/sectionContent.stories.tsx

```diff
@@ -0,0 +1,30 @@
+// Core dependencies
```

Comment:
How do I view this in Storybook? The component list I am seeing is:

<img width="230" alt="Screen Shot 2022-06-22 at 9 36 20 AM" src="https://user-images.githubusercontent.com/2446424/175089632-6347fd8d-9f54-46df-ac0d-a4cc498b392a.png">